<?xml version="1.0"?>

<launch>

	<!-- Network --> 
	<arg name="net_port" default="12346"/>
	<arg name="hostname" default="127.0.0.1"/>
	
	<!-- Position --> 
	<arg name="out_pose_topic" default="slam_out_pose"/>
	<arg name="pose_subscriber_queue_size" default="5"/>

	<!-- GDB -->
	<arg name="launch-prefix" default="xterm -e gdb --args"/>
  	
  	<node pkg="slam_bridge" type="pose_bridge" name="pose_bridge" output="screen">
    	<!-- Network --> 
		<param name="net_port" value="$(arg net_port)"/>
    	<param name="hostname" value="$(arg hostname)"/>
	    
	    <!-- Position subscription --> 
	    <param name="out_pose_topic" value="$(arg out_pose_topic)"/>
	    <param name="pose_subscriber_queue_size" value="$(arg pose_subscriber_queue_size)"/>
	</node>

</launch>
